calc-lexico.l

```diff
@@ -1,9 +1,8 @@
 %{
-#include <stdlib.h>
-#include <stdio.h>
-#include "structs.h"
+#include "structs.c"
```

Comment:
We really should include "structs.h" instead of any .c file

Reply:
solved